definitions/npm/lodash_v4.x.x/flow_v0.47.x-/test_lodash-v4.x.x.js

```diff
@@ -329,43 +321,58 @@ directStrings = _.map(nums, function(num) {
   return JSON.stringify(num);
 });
 
-var obj = {a:1, b:2};
+var obj = { a: 1, b: 2 };
 bool = _.conformsTo(obj, {
-  a: function(x:number) {
+  a: function(x: number) {
     return true;
-  },
+  }
 });
 
 num = _.defaultTo(undefined, 2);
-string = _.defaultTo(undefined, 'str');
-bool = _.defaultTo(true, 'str');
-string = _.defaultTo('str', true);
+string = _.defaultTo(undefined, "str");
+bool = _.defaultTo(true, "str");
+string = _.defaultTo("str", true);
 
-num = _.tap(1, function(n) { return false; });
-bool = _.thru(1, function(n) { return false; });
+num = _.tap(1, function(n) {
+  return false;
+});
+bool = _.thru(1, function(n) {
+  return false;
+});
 
 var timesNums: number[];
 
 timesNums = _.times(5);
 // $ExpectError string. This type is incompatible with number
-var strings : string[] = _.times(5);
-timesNums = _.times(5, function(i: number) { return i + 1; });
+var strings: string[] = _.times(5);
+timesNums = _.times(5, function(i: number) {
+  return i + 1;
+});
 // $ExpectError string. This type is incompatible with number
-timesNums = _.times(5, function(i: number) { return JSON.stringify(i); });
+timesNums = _.times(5, function(i: number) {
```

Comment:
Try putting this back on a single line, it might fix the build error.

Reply:
Thanks for pointing this out, I missed the failure. Putting it on one line isn't an option due to prettier, but I did move the comment to the right place.